data-showcase/grails-app/controllers/nl/thehyve/datashowcase/UrlMappings.groovy

```diff
@@ -22,7 +22,8 @@ class UrlMappings {
         "/api/projects"(controller: 'project') {
             action = [GET: 'index', POST: 'search']
         }
-        "/api/keywords"(controller: 'keyword', includes: ['index'])
+        "/api/keywords/$conceptCode"(controller: 'keyword', action: 'show')
```

Comment:
Suggestion: maybe use a route like `/api/keywords/by_concept/$conceptCode` to make clear that the last part of the url is not identifying a keyword, but a concept.